pennylane/devices/qubit_mixed/apply_operation.py

```diff
@@ -334,32 +348,25 @@ def _(op: type_op, state, is_state_batched=False, **kwargs):
     >>> state[0][0] = 1
```

Comment:
The state has 4 dims
```suggestion
    >>> state[0][0][0][0] = 1
```